src/lib.rs

```diff
@@ -765,6 +780,7 @@ impl GooseAttack {
             configuration,
             run_time: 0,
             attack_mode: GooseMode::Undefined,
+            scheduler: GooseScheduler::RoundRobin,
```

Comment:
Yes, I like that. And maybe as a followup we should add a GooseTaskScheduler with the same options (currently it uses the random method, but it would be nice for it also to be configurable). Fixed.